File: examples/swift/BackgroundRemoval.swift
import AILabTools
import Foundation

let client = try AILabToolsClient(
    apiKey: ProcessInfo.processInfo.environment["AILAB_API_KEY"]!
)

let result = try await client.background.remove(
    CutoutUniversalBackgroundRemovalParams(
        image: .fileURL(URL(fileURLWithPath: "./image.jpg")),
        returnForm: "whiteBK"
    )
)

print(result.data?.imageUrl ?? "")
client.close()

File: examples/kotlin/BackgroundRemoval.kt
import com.ailabtools.sdk.AILabClient
import com.ailabtools.sdk.FileInput
import com.ailabtools.sdk.generated.params.CutoutUniversalBackgroundRemovalParams
import java.io.File
import kotlinx.coroutines.runBlocking

fun main(): Unit = runBlocking {
    val client = AILabClient(
        apiKey = requireNotNull(System.getenv("AILAB_API_KEY")),
    )
    try {
        val result = client.background.remove(
            CutoutUniversalBackgroundRemovalParams(
                image = FileInput.fromFile(File("image.jpg"), "image/jpeg"),
                returnForm = "whiteBK",
            ),
        )
        println(result.data?.imageUrl)
    } finally {
        client.close()
    }
}
